_data/payments.yml

```diff
@@ -135,4 +135,4 @@ websites:
       sms: Yes
       software: Yes
       hardware: Yes
-      doc: https://money.yandex.ru/doc.xml?id=524852
+      doc: https://money.yandex.ru/doc.xml?id=524852
```

Comment:
As per our `.editorconfig`, it contains `insert_final_newline = true`.
Please respect that.